packages/web-components/src/card/card.ts

```diff
@@ -0,0 +1,207 @@
+import { attr, css, ElementStyles, observable } from '@microsoft/fast-element';
+import { FASTCard, StartEndOptions } from '@microsoft/fast-foundation';
+import { keyEnter, keySpace } from '@microsoft/fast-web-utilities';
+import { CardAppearance, CardControlSize, CardOrientation } from './card.options.js';
+
+/**
+ * Card configuration options
+ * @public
+ */
+export type CardOptions = StartEndOptions<Card>;
+
+/**
+ * @class Card component
+ *
+ * @remarks
+ * This class extends the FASTCard. a flexible content container
+ */
+export class Card extends FASTCard {
+  public connectedCallback(): void {
+    super.connectedCallback();
+    this.setInteractive();
+    this.updateComputedStylesheet();
+  }
+
+  public disconnectedCallback(): void {
+    super.disconnectedCallback();
+  }
+
+  /**
+   * Stores the computed stylesheet for the card
+   */
+  private computedStylesheet?: ElementStyles;
+
+  /**
+   * @property orientation;
+   * @default filled
+   * @remarks
+   * Determines the orientation of the card
+   */
+  @attr
+  public appearance?: CardAppearance;
+
+  /**
+   * @property orientation;
+   * @default vertical
+   * @remarks
+   * Determines the orientation of the card
+   */
+  @attr
+  public orientation?: CardOrientation;
+
+  /**
+   * @property controlSize
+   * @default medium
+   * @remarks
+   * Determines the size of the card
+   */
+  @attr({ attribute: 'control-size' })
+  public controlSize?: CardControlSize;
+
+  /**
+   * @property interactive
+   * @default false
+   * @remarks
+   * Determines whether card is interactable
+   */
+  @observable
+  @attr({ mode: 'boolean' })
+  public interactive: boolean = false;
```

Comment:
`selectable` appears to be the name for v9 - is there a reason to deviate from that?

Reply:
This came from the design spec which makes a differentiation between selectable and interactive cards. Interactive cards acting more like a button. But considering the react v9 card doesn't implement an "interactive" card and this component is already an a11y nightmare I think removing that feature is a good idea.

I've made the changes.